src/modules/products/controllers/search.controller.ts

```
@@ -0,0 +1,27 @@
+import { Controller, Get, Query, ValidationPipe, BadRequestException } from "@nestjs/common";
+import { SearchService } from "../services/search.service";
+import { SearchDto } from "../dtos/search.dto";
+import { FilterDto } from "../dtos/filter.dto";
+
+@Controller("products")
+export class SearchController {
+  constructor(private readonly searchService: SearchService) {}
+
+  @Get("search")
```

ESLint

[error] Replace `"search"` with `'search'`

(prettier/prettier)